x-pack/plugins/observability_onboarding/public/plugin.ts

```diff
@@ -80,9 +78,6 @@ export class ObservabilityOnboardingPlugin
     // and go to /app/observabilityOnboarding
     if (isObservabilityOnboardingUiEnabled) {
       core.application.register({
-        navLinkStatus: isServerlessEnabled
```

Comment:
This logic used to make the app searchable in serverless and not searchable in stasteful. Is that the behaviour we want to keep?